openapi/openapi.json

```diff
@@ -3797,7 +3797,7 @@
     "termsOfService": "https://www.pagopa.gov.it/",
-    "version": "1.1.19"
+    "version": "1.1.19-2-PAGOPA-2755-fix-fdr-3006"
```

Comment:
Is it right that this is the only OpenAPI update?

Reply:
I have not modified any interfaces, so it is right.